Assets/MixedRealityToolkit/_Core/Definitions/SpatialAwareness/MixedRealitySpatialAwarenessProfile.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class MixedRealitySpatialAwarenessProfile : MonoBehaviour {
```

Comment:
Scriptable object?

Reply:
it will be... forgot to block comment that out